public/js/components/CloseButton.js

```diff
@@ -4,13 +4,13 @@ const Svg = require("./utils/Svg");
 
 require("./CloseButton.css");
 
-function CloseButton({ handleClick }) {
-  return dom.div({ className: "close-btn", onClick: handleClick },
+function CloseButton({ handleClick, buttonClass }) {
+  return dom.div({ className: buttonClass !== undefined ? buttonClass : "close-btn", onClick: handleClick },
```

Comment:
should be additive
checkout classnames https://github.com/devtools-html/debugger.html/blob/master/public/js/components/Breakpoints.js#L84-L88